operator/config/overlays/cnsa/kustomization.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Adds namespace to all resources.
+namespace: ibm-spectrum-scale-csi-driver
+
+bases:
+- ../../manager/
```

Comment:
Shouldn't we just be building `default` and then overlay that?

Reply:
Oh, looks like the way you have default kustomization file, we shouldn't do that.

Reply:
So we should leave the default at config/default then have a separate overlay directory that lists the default as a resource?

Reply:
I thought you were gonna keep `config/default` in place when you mentioned overlays. But this is also fine I think.